src/genie/libs/parser/junos/show_chassis.py

```diff
@@ -381,15 +381,24 @@ class ShowChassisHardwareSchema(MetaParser):
         "chassis": {
             Optional("@junos:style"): str,
             "chassis-module": [
+                {
```

Comment:
This is updating a comment, I believe you meant to update the actual schema? Ideally these comments should be removed. They were added in the past because the schema engine did not directly support a list of something.